src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -0,0 +1,75 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+
+import { BuilderLayers, useCheckIfNewApplet } from 'shared/hooks';
+import { builderSessionStorage, Path } from 'shared/utils';
+import { applet } from 'shared/state';
+
+import {
+  appletActivitiesMocked,
+  appletActivityFlowsMocked,
+  appletDataMocked,
+  appletPasswordMocked,
+} from './mock';
```

Comment:
do we need mocked values anymore? maybe it's good idea to get rid of them?

Reply:
yes, we need the mocked data for a while till the BE will fix the issue with activities https://mindlogger.atlassian.net/browse/M2-1139 and will add a separate API to retrieve items https://mindlogger.atlassian.net/browse/M2-1125?